utils/prompts.py

```diff
@@ -406,6 +406,10 @@ def get_dataset(
     corrupted_tokens = model.to_tokens(
         all_prompts[1:] + [all_prompts[0]], prepend_bos=True
     ).to(device)
+    assert (clean_tokens[:, -1] != model.tokenizer.bos_token_id).all(), (
```

Comment:
@curt-tigges I've added this valuable assertion check!

Reply:
Excellent idea